armi/bookkeeping/visualization/entryPoint.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+import sys
+import pathlib
+
+from armi import runLog
+from armi.cli import entryPoint
+
+
+class VisFileEntryPoint(entryPoint.EntryPoint):
+    """
+    Create visualization files from database files.
+    """
+
+    name = "vis-file"
+    description = "Convert ARMI databases in to visualization files"
+
+    _FORMAT_VTK = "vtk"
+    _SUPPORTED_FORMATS = {_FORMAT_VTK}
+
+    def __init__(self):
+        entryPoint.EntryPoint.__init__(self)
+
+        self.nodes = None
```

Comment:
nodes here without a docstring is confusing and I had to scroll down to figure out what it was. Can you add a 

```
self.nodes = None
"""Data base nodes to visualize"""
``` 
kinda docstring?